lib/manager/pipenv/artifacts.ts

```diff
@@ -28,14 +48,15 @@ export async function updateArtifacts({
     const localPipfileFileName = join(config.localDir, pipfileName);
     await outputFile(localPipfileFileName, newPipfileContent);
     const localLockFileName = join(config.localDir, lockFileName);
-    const cmd = 'pipenv lock';
+    const cmd = ['pip install pipenv', 'pipenv lock'];
     const execOptions: ExecOptions = {
       extraEnv: {
         PIPENV_CACHE_DIR: cacheDir,
       },
       docker: {
         image: 'renovate/pipenv',
         volumes: [cacheDir],
+        tagConstraint: getPythonConstraint(newPipfileContent, config),
```

Comment:
So this is undefined if not found, right?

Reply:
yes